zappa/zappa.py

```diff
@@ -809,24 +809,30 @@ def create_api_gateway_routes(self, lambda_arn, api_name=None, api_key_required=
             self.create_and_setup_cors(restapi, resource, invocations_uri, 1, cors_options)  # pragma: no cover
         return restapi
 
-    def create_authorizer(self, restapi, uri, identity_source, authorizer_result_ttl, identity_validation_expression):
+    def create_authorizer(self, restapi, uri, authorizer):
```

Comment:
`create_authorizer` is only called from `create_api_gateway_routes` so I decided to change the signature (args). Nothing else should break.